packages/build/tests/install/snapshots/tests.js.md

```diff
@@ -755,54 +761,43 @@ Generated by [AVA](https://ava.li).
     > Flags␊
       debug: true␊
       featureFlags: {}␊
-      repositoryRoot: /file/path␊
+      repositoryRoot: packages/build/tests/install/fixtures/local_missing␊
       testOpts:␊
         pluginsListUrl: test␊
         silentLingeringProcesses: true␊
     ␊
     > Current directory␊
-      /file/path␊
+      packages/build/tests/install/fixtures/local_missing␊
     ␊
     > Config file␊
-      /file/path␊
+      packages/build/tests/install/fixtures/local_missing/netlify.toml␊
     ␊
     > Resolved config␊
       plugins:␊
         - inputs: {}␊
           origin: config␊
-          package: /file/path␊
+          package: plugin/main.js␊
     ␊
     > Context␊
       production␊
     ␊
     > Loading plugins␊
-       - /file/path from netlify.toml␊
+       - plugin/main.js@1.0.0 from netlify.toml␊
     ␊
     ────────────────────────────────────────────────────────────────␊
-      Plugin "/file/path" internal error                      ␊
+      Plugin "plugin/main.js" internal error                      ␊
     ────────────────────────────────────────────────────────────────␊
     ␊
       Error message␊
       Error: Could not import plugin:␊
     ␊
     ␊
-      Plugin details␊
-      Package:        /file/path␊
-      Version:        1.0.0␊
-      Repository:     test␊
-    ␊
-      Error location␊
-      While loading "/file/path" from netlify.toml␊
-    STACK TRACE␊
-    ␊
-      Error properties␊
-      { code: 'MODULE_NOT_FOUND' }␊
```

Comment:
We're missing this plugin details error, is it expected?

Reply:
I have a fix for this ready to commit.

Reply:
Fixed in https://github.com/netlify/build/pull/2456/commits/a88ab2fc336fcebd61107ed99c2ecc0a4303d7d9.